deploy/charts/external-secrets/templates/validatingwebhook.yaml

```diff
@@ -50,10 +48,8 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: externalsecret-validate
   labels:
+    {{- include "external-secrets-webhook.labels" . | nindent 4 }}
     external-secrets.io/component: webhook
-    {{- with .Values.commonLabels }}
-    {{ toYaml . | nindent 4 }}
-    {{- end }}
```

Comment:
we should probably still allow for commonLabels to be applied here.

Reply:
The `external-secrets-webhook.labels` already includes the common labels see https://github.com/external-secrets/external-secrets/blob/main/deploy/charts/external-secrets/templates/_helpers.tpl#L66